tests/python/python-couchbase-empty-password-python-test.yml

```
@@ -0,0 +1,23 @@
+id: python-couchbase-empty-password-python
+valid:
+  - |
+    import os
+    from couchbase.cluster import Cluster, ClusterOptions
+    from couchbase_core.cluster import PasswordAuthenticator
+    PasswordAuthenticator('username', os.env['pass'])
+  - |
+    import os
+    from couchbase.cluster import Cluster, ClusterOptions
+    from couchbase_core.cluster import PasswordAuthenticator 
```

yamllint

[error] trailing spaces

(trailing-spaces)